webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,44 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+
+    updateConversationMetadata: function (conversationMetadata) {
+        if (conversationMetadata) {
+            var metadataToSend = conversationMetadata.metadata;
+            if (
+                conversationMetadata.groupId &&
+                metadataToSend &&
+                kommunicateCommons.isObject(conversationMetadata) &&
```

Comment:
at 121 we are already checking if `conversationMetadata` exists, so maybe we can ignore this check
```js
kommunicateCommons.isObject(conversationMetadata) &&
```

Reply:
This is an object verification check.

Reply:
first, we are accessing a property assuming it is an object, here:
```js
var metadataToSend = conversationMetadata.metadata;
```
and later we are doing object validation.
```js
kommunicateCommons.isObject(conversationMetadata) &&
```
isn't little wrong and overhead?